joshprzybyszewski/cribbage
# Update UI to Material

## What broke / What you're adding
Changing over from the old stuff (tailwind) to our new found best friend: Material UI.

Here's a peek with the drawer pulled out:
![image](https://user-images.githubusercontent.com/19275178/87497597-7fdfa000-c61b-11ea-821b-cd5a32cee2b4.png)

- Changed all components to use Material UI components
- Reworked navbar
  - Navbar options now live in a drawer that pulls out from the left
  - Drawer contents differ depending on whether the user is logged in or not
  - "Login", "Register", "Logout" buttons all within the drawer now
- Added `Layout.jsx`, which handles the app skeleton
- Changed login and register forms to use Material

## How you did it
For the drawer, I followed (read: copied) the examples [here](https://material-ui.com/components/drawers/). For the other stuff, I followed various examples from the Material UI doc for various components (most notably, [forms](https://material-ui.com/components/text-fields/)).

## How to test it and how to try to break it
Stand up the server, start the client. See the new beautiful (albeit very google-y) UI. Try the drawer. See that it changes if you're logged in. See that the login/logout and navigation buttons work.
